### PR TITLE
cli: add seed to add bank command

### PR DESCRIPTION
**Pull request by exdx**
Updates the `mfi group add bank` command to have an optional `seed` boolean argument. If `seed` is set then the command generates a PDA and uses it to sign the ix on behalf of the bank. 

Based off the work in https://github.com/mrgnlabs/marginfi-v2/issues/154. 

Generates a PDA from seeds and a random string iteratively. Uses the PDA to then sign on behalf of the bank in the `group_add_bank` command. 

Continuation of #158

### CODE REVIEW

**Comment by jkbpvsc** [comment]
Seeing lint and tests failing

**Comment by exdx** [comment]
CI is green -- I accidentally feature-gated imports in `utils.rs` which broke a lot of things lol. 

Ready for another look. I think conceptually this PR is ready I just want to make sure the business logic is correct. 

I do think using generics could slim this down further but I will look into that as a follow-up. Would be nice to abstract the different between `AddBank` and `AddBankWithSeed` further via a trait and only have one fn here instead of two. They are very similar conceptually.

File: clients/rust/marginfi-cli/src/processor/mod.rs
Comment (jkbpvsc) [inline clients/rust/marginfi-cli/src/processor/mod.rs:1]: So a few nuances.

To make sure we only use a single singer for the multisig, before we were using 3 singers (authority -- ms, fee_payer, and bank).

For the new instruction to work, we must make sure we don't sign with the `bank_keypair` or the `fee_payer`.
Only the authority will sign in the multisig.

So in code:

1. We have to get this without an explicit fee payer
```
let mut signing_keypairs = config.get_signers(true -> false);
``

2. We have to only include the `bank_keypair` when not using the `--seed`,
so removing this line
```
 signing_keypairs.push(&bank_keypair);
```

And here
```
   let message = Message::new(&add_bank_ixs, Some(&config.authority()));
```

**Review by jkbpvsc — CHANGES_REQUESTED** [review]

File: clients/rust/marginfi-cli/src/processor/mod.rs
Comment (exdx) [reply clients/rust/marginfi-cli/src/processor/mod.rs:1]: Ok sounds good!

**Review by jkbpvsc — APPROVED** [review]

**Pull request merged**